modules/openapi-generator/src/main/resources/r/model.mustache

```diff
@@ -212,19 +213,30 @@
       self$`{{baseName}}` <- vector("list", length = nrow(data.frame))
       for (row in 1:nrow(data.frame)) {
           {{baseName}}.node <- {{dataType}}$new()
-          {{baseName}}.node$fromJSON(jsonlite::toJSON(data.frame[row,,drop = TRUE], auto_unbox = TRUE))
+          {{baseName}}.node$fromJSON(jsonlite::toJSON(data.frame[row,,drop = F], auto_unbox = TRUE))
```

Comment:
Do you mind explaining why change drop from TRUE to F (false)?

Reply:
@wing328 drop = TRUE is actually making a data.frame of one column reducing the dimension and making it as list
and later when it is being converted back to object it is not being converted properly to original object as its dimension is reduced.

For Example: consider this json 
```
[{"id" : 1234}, {"id": 1122}, {"id" : 1024}]
```
when the above json is used with jsonlite with drop = TRUE the key name "id" is lost when it is converted from object to json and back to object.

Im not sure why the drop = TRUE is kept initially?

Reply:
If I remember correctly, it only works if I set it to TRUE.

I think we need to add some tests to ensure all previous use cases are covered.